### PR TITLE
[hi] Localize `Serverless` for Hindi

### PR DESCRIPTION
**Pull request by shivam-sharma7**
`Serverless` now Localized into Hindi.

Signed-off-by: Shivam Sharma <meshivam81@gmail.com>

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by shivam-sharma7** [comment]
review/ @NitishKumar06 @anubha-v-ardhan

**Comment by shivam-sharma7** [comment]
review/ @NitishKumar06 @anubha-v-ardhan

**Comment by shivam-sharma7** [comment]
review/ @Garima-Negi

File: content/hi/serverless.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+title: सर्वरलेस (Serverless)
+status: Completed
+category: प्रौद्योगिकी
+---
+
+### यह क्या है
+
+सर्वरलेस एक क्लाउड नेटिव डेवलपमेंट मॉडल है जो डेवलपर्स को सर्वर को प्रबंधित किए बिना एप्लिकेशन बनाने और चलाने की अनुमति देता है। सर्वर रहित सर्वर अभी भी हैं, लेकिन वे [abstracted](/abstraction/) ऐप डेवलपमेंट से दूर हैं। एक क्लाउड प्रदाता सर्वर इन्फ्रास्ट्रक्चर के प्रावधान, रखरखाव और [scaling](/scalability/) के नियमित कार्य को संभालता है। डेवलपर्स केवल तैनाती के लिए अपने कोड को [containers](/container/) में पैकेज कर सकते हैं। एक बार तैनात होने के बाद, सर्वर रहित ऐप्स मांग का जवाब देते हैं और आवश्यकतानुसार स्वचालित रूप से ऊपर और नीचे स्केल करते हैं। सार्वजनिक क्लाउड प्रदाताओं से सर्वर रहित पेशकशों को आमतौर पर इवेंट-संचालित निष्पादन मॉडल के माध्यम से ऑन-डिमांड मीटर किया जाता है। नतीजतन, जब कोई सर्वर   रहित फ़ंक्शन निष्क्रिय रहता है, तो इसका कुछ भी खर्च नहीं होता है।
+
+### समस्या यह संबोधित करता है
```
Comment (NitishKumar06) [inline content/hi/serverless.md:11]:
```suggestion
### समस्या जो यह संबोधित करता है
```

**Review by NitishKumar06 — COMMENTED** [review]
Please add this

**Comment by shivam-sharma7** [comment]
review/ @Garima-Negi

**Review by NitishKumar06 — COMMENTED** [review]
Please squash your commits

File: content/hi/serverless.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: सर्वरलेस (Serverless)
+status: Completed
+category: प्रौद्योगिकी
+---
+
+### यह क्या है
```
Comment (anubha-v-ardhan) [inline content/hi/serverless.md:7]:
```suggestion
## यह क्या है
```

File: content/hi/serverless.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+सर्वरलेस एक क्लाउड नेटिव डेवलपमेंट मॉडल है जो डेवलपर्स को सर्वर को प्रबंधित किए बिना एप्लिकेशन बनाने और चलाने की अनुमति देता है। सर्वर रहित सर्वर अभी भी हैं, लेकिन वे [abstracted](/abstraction/) ऐप डेवलपमेंट से दूर हैं। एक क्लाउड प्रदाता सर्वर इन्फ्रास्ट्रक्चर के प्रावधान, रखरखाव और [scaling](/scalability/) के नियमित कार्य को संभालता है। डेवलपर्स केवल तैनाती के लिए अपने कोड को [containers](/container/) में पैकेज कर सकते हैं। एक बार तैनात होने के बाद, सर्वर रहित ऐप्स मांग का जवाब देते हैं और आवश्यकतानुसार स्वचालित रूप से ऊपर और नीचे स्केल करते हैं। सार्वजनिक क्लाउड प्रदाताओं से सर्वर रहित पेशकशों को आमतौर पर इवेंट-संचालित निष्पादन मॉडल के माध्यम से ऑन-डिमांड मीटर किया जाता है। नतीजतन, जब कोई सर्वर   रहित फ़ंक्शन निष्क्रिय रहता है, तो इसका कुछ भी खर्च नहीं होता है।
+
+### समस्या जो यह संबोधित करता है
```
Comment (anubha-v-ardhan) [inline content/hi/serverless.md:11]:
```suggestion
## समस्या
```

File: content/hi/serverless.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+एक मानक [Infrastructure-as-a-Service (IaaS)](/infrastructure-as-a-service/) [cloud computing](/cloud-computing/) मॉडल के तहत, उपयोगकर्ता क्षमता की इकाइयों को पूर्व-खरीद करते हैं, जिसका अर्थ है कि आप अपने ऐप्स चलाने के लिए हमेशा-ऑन सर्वर घटकों के लिए सार्वजनिक क्लाउड प्रदाता का भुगतान करते हैं। उच्च मांग के समय सर्वर क्षमता को बढ़ाना और उस क्षमता की आवश्यकता नहीं होने पर इसे कम करना उपयोगकर्ता की जिम्मेदारी है। ऐप को चलाने के लिए आवश्यक क्लाउड इन्फ्रास्ट्रक्चर तब भी सक्रिय रहता है, जब ऐप का उपयोग नहीं किया जा रहा हो।
+
+### यह  कैसे मदद करता है
```
Comment (anubha-v-ardhan) [inline content/hi/serverless.md:15]:
```suggestion
## समाधान
```

File: content/hi/serverless.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+title: सर्वरलेस (Serverless)
+status: Completed
+category: प्रौद्योगिकी
+---
```
Comment (anubha-v-ardhan) [inline content/hi/serverless.md:5]:
```suggestion
---
title: सर्वरलेस (Serverless)
status: Completed
category: प्रौद्योगिकी
tags: ["architecture", "", ""]
---
```

File: content/hi/serverless.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+### यह क्या है
+
+सर्वरलेस एक क्लाउड नेटिव डेवलपमेंट मॉडल है जो डेवलपर्स को सर्वर को प्रबंधित किए बिना एप्लिकेशन बनाने और चलाने की अनुमति देता है। सर्वर रहित सर्वर अभी भी हैं, लेकिन वे [abstracted](/abstraction/) ऐप डेवलपमेंट से दूर हैं। एक क्लाउड प्रदाता सर्वर इन्फ्रास्ट्रक्चर के प्रावधान, रखरखाव और [scaling](/scalability/) के नियमित कार्य को संभालता है। डेवलपर्स केवल तैनाती के लिए अपने कोड को [containers](/container/) में पैकेज कर सकते हैं। एक बार तैनात होने के बाद, सर्वर रहित ऐप्स मांग का जवाब देते हैं और आवश्यकतानुसार स्वचालित रूप से ऊपर और नीचे स्केल करते हैं। सार्वजनिक क्लाउड प्रदाताओं से सर्वर रहित पेशकशों को आमतौर पर इवेंट-संचालित निष्पादन मॉडल के माध्यम से ऑन-डिमांड मीटर किया जाता है। नतीजतन, जब कोई सर्वर   रहित फ़ंक्शन निष्क्रिय रहता है, तो इसका कुछ भी खर्च नहीं होता है।
```
Comment (anubha-v-ardhan) [inline content/hi/serverless.md:9]:
```suggestion
सर्वरलेस एक क्लाउड नेटिव डेवलपमेंट मॉडल है जो डेवलपर्स को सर्वर को प्रबंधित किए बिना एप्लिकेशन बनाने और चलाने की अनुमति देता है। सर्वरलेस में सर्वर अभी भी हैं, लेकिन वे [ऐब्स्ट्रैक्टेड](/hi/abstraction/) ऐप डेवलपमेंट से दूर हैं। एक क्लाउड प्रदाता सर्वर इन्फ्रास्ट्रक्चर के प्रावधान, रखरखाव और [स्केलिंग](/hi/scalability/) के नियमित कार्य को संभालता है। डेवलपर्स डिप्लॉयमेंट के लिए अपने कोड को [कंटेनर](/hi/container/) में पैकेज कर सकते हैं। एक बार डिप्लॉय होने के बाद, सर्वरलेस ऐप्स डिमांड का जवाब देते हैं और आवश्यकतानुसार स्वचालित रूप से ऊपर और नीचे स्केल करते हैं। सार्वजनिक क्लाउड प्रदाताओं से सर्वरलेस पेशकशों को आमतौर पर इवेंट-संचालित निष्पादन मॉडल के माध्यम से ऑन-डिमांड मीटर किया जाता है। नतीजतन, जब कोई सर्वरलेस फ़ंक्शन निष्क्रिय रहता है, तो इसका कुछ भी खर्च नहीं होता है।
```

File: content/hi/serverless.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+### समस्या जो यह संबोधित करता है
+
+एक मानक [Infrastructure-as-a-Service (IaaS)](/infrastructure-as-a-service/) [cloud computing](/cloud-computing/) मॉडल के तहत, उपयोगकर्ता क्षमता की इकाइयों को पूर्व-खरीद करते हैं, जिसका अर्थ है कि आप अपने ऐप्स चलाने के लिए हमेशा-ऑन सर्वर घटकों के लिए सार्वजनिक क्लाउड प्रदाता का भुगतान करते हैं। उच्च मांग के समय सर्वर क्षमता को बढ़ाना और उस क्षमता की आवश्यकता नहीं होने पर इसे कम करना उपयोगकर्ता की जिम्मेदारी है। ऐप को चलाने के लिए आवश्यक क्लाउड इन्फ्रास्ट्रक्चर तब भी सक्रिय रहता है, जब ऐप का उपयोग नहीं किया जा रहा हो।
```
Comment (anubha-v-ardhan) [inline content/hi/serverless.md:13]:
```suggestion
एक मानक [इन्फ्रास्ट्रक्चर एस सर्विस (IaaS)](/infrastructure-as-a-service/) [क्लाउड कंप्यूटिंग](/cloud-computing/) मॉडल के तहत, उपयोगकर्ता क्षमता की इकाइयों को पूर्व-खरीद करते हैं, जिसका अर्थ है कि आप अपने ऐप्स चलाने के लिए हमेशा-ऑन सर्वर घटकों के लिए सार्वजनिक क्लाउड प्रदाता का भुगतान करते हैं। उच्च मांग के समय सर्वर क्षमता को बढ़ाना और उस क्षमता की आवश्यकता नहीं होने पर इसे कम करना उपयोगकर्ता की जिम्मेदारी है। ऐप को चलाने के लिए आवश्यक क्लाउड इन्फ्रास्ट्रक्चर तब भी सक्रिय रहता है, जब ऐप का उपयोग नहीं किया जा रहा हो।
```

File: content/hi/serverless.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+
+### यह  कैसे मदद करता है
+
+सर्वर रहित आर्किटेक्चर के साथ, इसके विपरीत, ऐप्स केवल आवश्यकतानुसार लॉन्च किए जाते हैं। जब कोई ईवेंट चलने के लिए ऐप कोड को ट्रिगर करता है, तो सार्वजनिक क्लाउड प्रदाता गतिशील रूप से उस कोड के लिए संसाधन आवंटित करता है। कोड निष्पादित होने पर उपयोगकर्ता भुगतान करना बंद कर देता है। लागत और दक्षता लाभों के अलावा, सर्वर रहित डेवलपर्स को ऐप स्केलिंग और सर्वर प्रोविजनिंग से जुड़े नियमित और छोटे कार्यों से मुक्त करता है। सर्वर रहित होने के साथ, नियमित कार्य जैसे ऑपरेटिंग सिस्टम और फ़ाइल सिस्टम का प्रबंधन, सुरक्षा पैच, लोड संतुलन, क्षमता प्रबंधन, स्केलिंग, लॉगिंग और निगरानी सभी क्लाउड सेवा प्रदाता को लोड कर दिए जाते हैं।
```
Comment (anubha-v-ardhan) [inline content/hi/serverless.md:17]:
```suggestion
इसके विपरीत, सर्वरलेस आर्किटेक्चर के साथ, ऐप्स केवल आवश्यकतानुसार लॉन्च किए जाते हैं। जब कोई ईवेंट चलने के लिए ऐप कोड को ट्रिगर करता है, तो सार्वजनिक क्लाउड प्रदाता गतिशील रूप से उस कोड के लिए संसाधन आवंटित करता है। कोड निष्पादित होने पर उपयोगकर्ता भुगतान करना बंद कर देता है। लागत और दक्षता लाभों के अलावा, सर्वरलेस डेवलपर्स को ऐप स्केलिंग और सर्वर प्रोविजनिंग से जुड़े नियमित और छोटे कार्यों से मुक्त करता है। सर्वरलेस होने के साथ, नियमित कार्य जैसे ऑपरेटिंग सिस्टम और फ़ाइल सिस्टम का प्रबंधन, सुरक्षा पैच, लोड संतुलन, क्षमता प्रबंधन, स्केलिंग, लॉगिंग और निगरानी सभी क्लाउड सेवा प्रदाता को लोड कर दिए जाते हैं।
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

**Review by anubha-v-ardhan — APPROVED** [review]
LGTM!

**Review by jayesh-srivastava — APPROVED** [review]
LGTM!

**Comment by shivam-sharma7** [comment]
Thanks

**Pull request merged**